src/database/entities/user.entity.ts

```diff
@@ -0,0 +1,106 @@
+import { Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+
+@Entity('auth.users')
+export class UserEntity {
+  @PrimaryGeneratedColumn('uuid')
+  readonly id: string;
+
+  @Column({ nullable: true })
+  instance_id: string;
+
+  @Column()
+  aud: string;
+
+  @Column({ nullable: true })
+  role: string;
+
+  @Column()
+  email: string;
+
+  @Column()
+  encrypted_password: string;
+
+  @Column({ nullable: true })
+  email_confirmed_at: Date;
+
+  @Column({ nullable: true })
+  invited_at: Date;
+
+  @Column({ nullable: true })
+  confirmation_token: string;
+
+  @Column({ nullable: true })
+  confirmation_sent_at: Date;
+
+  @Column({ nullable: true })
+  recovery_token: string;
+
+  @Column({ nullable: true })
+  recovery_sent_at: Date;
+
+  @Column({ nullable: true })
+  email_change_token_new: string;
+
+  @Column({ nullable: true })
+  email_change: string;
+
+  @Column({ nullable: true })
+  email_change_sent_at: Date;
+
+  @Column({ nullable: true })
+  last_sign_in_at: Date;
+
+  @Column({ type: 'jsonb', nullable: true })
+  raw_app_meta_data: Record<string, any>;
+
+  @Column({ type: 'jsonb', nullable: true })
```

Comment:
É jsonb mesmo?

Reply:
CREATE TABLE
auth.users (
instance_id uuid NULL,
id uuid NOT NULL,
aud character varying(255) NULL,
role character varying(255) NULL,
email character varying(255) NULL,
encrypted_password character varying(255) NULL,
email_confirmed_at timestamp with time zone NULL,
invited_at timestamp with time zone NULL,
confirmation_token character varying(255) NULL,
confirmation_sent_at timestamp with time zone NULL,
recovery_token character varying(255) NULL,
recovery_sent_at timestamp with time zone NULL,
email_change_token_new character varying(255) NULL,
email_change character varying(255) NULL,
email_change_sent_at timestamp with time zone NULL,
last_sign_in_at timestamp with time zone NULL,
raw_app_meta_data jsonb NULL,
raw_user_meta_data jsonb NULL,
is_super_admin boolean NULL,
created_at timestamp with time zone NULL,
updated_at timestamp with time zone NULL,
phone text NULL DEFAULT NULL::character varying,
phone_confirmed_at timestamp with time zone NULL,
phone_change text NULL DEFAULT ''::character varying,
phone_change_token character varying(255) NULL DEFAULT ''::character varying,
phone_change_sent_at timestamp with time zone NULL,
confirmed_at timestamp with time zone NULL DEFAULT LEAST(email_confirmed_at, phone_confirmed_at),
email_change_token_current character varying(255) NULL DEFAULT ''::character varying,
email_change_confirm_status smallint NULL DEFAULT 0,
banned_until timestamp with time zone NULL,
reauthentication_token character varying(255) NULL DEFAULT ''::character varying,
reauthentication_sent_at timestamp with time zone NULL,
is_sso_user boolean NOT NULL DEFAULT false,
deleted_at timestamp with time zone NULL
);

ALTER TABLE
auth.users
ADD
CONSTRAINT users_pkey PRIMARY KEY (id)